File: Task/Shoelace-formula-for-polygonal-area/Python/shoelace-formula-for-polygonal-area-1.py
>>> def area_by_shoelace(x, y):
    "Assumes x,y points go around the polygon in one direction"
    return abs( sum(i * j for i, j in zip(x,             y[1:] + y[:1]))
               -sum(i * j for i, j in zip(x[1:] + x[:1], y            ))) / 2

>>> points = [(3,4), (5,11), (12,8), (9,5), (5,6)]
>>> x, y = zip(*points)
>>> area_by_shoelace(x, y)
30.0
>>>

File: Task/Shoelace-formula-for-polygonal-area/Julia/shoelace-formula-for-polygonal-area-1.jl
"""
Assumes x,y points go around the polygon in one direction.
"""
shoelacearea(x, y) =
			abs(sum([i * j for (i, j) in zip(x, append!(y[2:end], y[1]))]) -
				sum([i * j for (i, j) in zip(append!(x[2:end], x[1]), y)])) / 2

x, y = [3, 5, 12, 9, 5], [4, 11, 8, 5, 6]
@show x y shoelacearea(x, y)
